packs/jira/aliases/create_issue.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: "create_jira_issue"
+action_ref: "jira.create_issue"
+description: "Create a JIRA issue."
+formats:
+    - "create issue with summary {{summary}} description {{description}} of type {{type}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}} priority {{priority}}"
```

Comment:
Yeah. Definitely not the way to go, especially considering you can have arbitrary key=value pairs at the end, so this waterfall is redundant.

@lakshmi-kannan you're probably new to StackStorm, but you should try to read and understand the documentation before contributing ❤️